### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by glennc**
Change the readme to point to docs.

@muratg @ahmetalpbalkan 

helps #118

### CODE REVIEW

File: 1.0.0-rc1-update1/README.md [hunk 1]
```diff
@@ -41,4 +41,6 @@ This image provides the following environment variables:
 In addition to these, `PATH` is set to include the `dnx`/`dnu` executables.
 
 [home]: https://github.com/aspnet/home
+[using-docker-article]: https://docs.asp.net/en/latest/getting-started/installing-on-linux.html#using-docker
 [webdev-article]: http://blogs.msdn.com/b/webdev/archive/2015/01/14/running-asp-net-5-applications-in-linux-containers-with-docker.aspx
```
Comment (ahmetalpbalkan) [inline 1.0.0-rc1-update1/README.md]: unused link

**Comment by muratg** [comment]
@glennc merge?

**Comment by ahmetalpbalkan** [comment]
@glennc ping. Also when you update please make sure Docker Hub page gets updated too.

**Comment by muratg** [comment]
@glennc this is not valid anymore. We should still update the readme, though so could you send a new one?

**Pull request closed without merging**